site/gatsby-site/package.json

```diff
@@ -106,7 +107,9 @@
     "test:e2e:ci": "start-server-and-test start http://localhost:8000 cypress:run"
   },
   "devDependencies": {
-    "@netlify/plugin-gatsby": "^3.6.2",
+    "@netlify/plugin-gatsby": "^3.5.2",
```

Comment:
We should keep the greatest version `^3.6.2` of `@netlify/plugin-gatsby` package

Reply:
Alright, I changed @netlify/plugin-gatsby back to ^3.6.2.